DigitalLearningSolutions.Web/Styles/login/chooseACentre.scss

```diff
@@ -0,0 +1,13 @@
+﻿@import "~nhsuk-frontend/packages/core/all";
+
+.centre-role-tag {
+    text-align: center;
+    margin-right: 5px;
+}
+
+.choose-centre-td {
+  @include mq($until: desktop) {
```

Comment:
Is there a reason you're using "mq" over "govuk-media-query" syntax (not sure if they're any different)? Also, I think !important might cause some issues in the future (as nothing can override it), so we could make use of the CSS specificity and do something like Steve has done in https://github.com/TechnologyEnhancedLearning/DLSV2/pull/281/files#
(check out welcome.scss L16 - class in a class is more specific than just a class so CSS will use that style when deciding between the two)